test/extended/testdata/deployments/deployment-history-limit.yaml

```diff
@@ -12,7 +12,7 @@ spec:
         name: history-limit
     spec:
       containers:
-      - image: "docker.io/alpine:3.6"
+      - image: "docker.io/centos:centos7"
```

Comment:
I remember @kargakis had a PR to switch it to alpine.

I get that if we start downloading the fat centos we may reach some timeouts in test conditions but that might be better solved by adjusting the tests to be tolerant to image pulls as that can take arbitrary time for any image.

I am fine with centos7 as that image should be reused by all our tests so pulling it should hit the cache most of the times. If we could have centos7-atomic that would be even better.

Reply:
@tnozicka we use centos:centos7 in other deployment test fixtures so it should be already pulled on node. This one fixture use alpine forcing docker to pull that image from docker hub which cause delays. I don't care to what image we default in the end, but when we choose we should stick with it in all fixtures.

Reply:
Also openshift/origin image (which is for sure present on the nodes running tests) is based on centos:centos7, so theoretically we don't have to pull the layers, just tag existing layers.

Reply:
do we have pullIfNotPresent or alwaysPull by default? Is the openshift/origin guaranteed to be (freshly) build on top of latest centos7?

for some reason I though we are running from RPMs in our tests so openshift/origin would be there...

with that said I am for centos either way so it is consistent across fixtures